tests/consensus_tests/test_resharding_abort_live_lock.py

```
@@ -0,0 +1,34 @@
+
+from .fixtures import *
+from .test_resharding import all_replicas, bootstrap_resharding, migrate_points
+from .utils import *
+
+
+def test_resharding_abort_live_lock(tmp_path: pathlib.Path):
+    # Bootstrap resharding cluster
+    peer_uris, _ = bootstrap_resharding(tmp_path, direction="down")
+
+    # Trigger resharding transfers to last peer, to put replicas into `ReshardingScaleDown` state
+    info = get_collection_cluster_info(peer_uris[-1], "test_collection")
```

Ruff (0.14.10)

`get_collection_cluster_info` may be undefined, or defined from star imports

(F405)